OPERATOR.rst

```diff
@@ -313,6 +344,27 @@ For GitLab or ClamAV updates, use the ``--no-restart`` flag in order to leave
 the instance stopped after the snapshot has been created. There is no point in
 starting the instance only to have the update terminate it again.
 
+Update GitLab via ``yum update``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To apply updates to the Linux instance running GitLab, SSH into the system and
+use ``yum`` to apply the package updates. It is recommended to run the
+``screen`` command prior to applying any updates. This will create a session
+that will be recoverable should your connection drop while the updates are in
+progress::
```

Comment:
This has never happened to me. Also, if we do this periodically and keep the AMI fresh, there will never be that many updates. I would remove `screen`.